utils/file_helpers.py

```diff
@@ -0,0 +1,45 @@
+import zipfile
+import pandas as pd
+
+
+def zip_file_converter_to_df(zip_file, reportType) -> pd.DataFrame:
```

Comment:
```suggestion
def zip_file_converter_to_df(zip_file, reportType) -> Optional[pd.DataFrame:]
```
Так как функция также может возвращать `None`